packages/data/src/registry.js

```diff
@@ -279,5 +321,16 @@ export function createRegistry( storeConfigs = {}, parent = null ) {
 		parent.subscribe( globalListener );
 	}
 
-	return withPlugins( registry );
+	const storesMap = {
+		stores,
+		parent,
+	};
+	StoresMap.set( registry, storesMap );
```

Comment:
Do we really need this map, I mean can't we just have `__experimentalSubscribeStores` call the `__experimentalSubscribeStores` for  the parent registries when the store is not available on the current registry?

Reply:
Shoot! We can! I have no idea why I haven't thought about this way simpler solution 😅, thx!

Reply:
Pushed ecc8d74.